apps/user-profile/src/components/modules/UserOnboardingModal/OnboardingCard/OnboardingCard.js

```diff
@@ -12,8 +12,12 @@ export default function OnboardingCard({
     subtitle,
     completed,
 }) {
+    const [isActive, setIsActive] = useState(false);
```

Comment:
you can remove this logic once CSS applied

Reply:
Reverted.